lib/components/details/details.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useRef } from "react";
+import cn from "classnames";
+import { Details as DetailsJs } from "lbh-frontend";
+import { ReactElement } from "react-router/node_modules/@types/react";
+
+import "./styles.scss";
+
+interface DetailsProps {
+  title: string;
+  children: ReactElement;
+  className?: string;
+}
+
+export const Details = ({ title, children, className }: DetailsProps) => {
```

Comment:
If you look at some of the other components I've tried to make all the common components support forwarding refs to functional components. ErrorSummary might be a good one to look at

Reply:
The only reason I didn't for this component is that it specifically uses "details" as it's root element; though if we wanted we could wrap that in a component?